### PR TITLE
Using a CLI to generate the Python package

### LINKED ISSUE #71: Use the ``pydita-ast`` project as a CLI tool
Similarly to what we do at [ansys-templates](https://github.com/ansys/ansys-templates), we should also allow for CLI usage of this tool, such that you do not really have to run the ``xml2rst.py`` file by hand. Configurable options should also be accepted via CLI.

Have a look at how the CLI is enabled:
https://github.com/ansys/ansys-templates/blob/6fc1b57a468366f68a694b234520792edc737d67/pyproject.toml#L74

And the different options exposed:
https://github.com/ansys/ansys-templates/blob/main/src/ansys/templates/cli.py

### PR DESCRIPTION
**Pull request by clatapie**
Closes #71.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,6 +26,7 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
+    "click>=7.0,<9.0.0",
```
Comment (RobPasMue) [inline pyproject.toml:29]: Fix a certain click deps on your tests

File: src/pydita_ast/cli.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    pass
+
+
+@package.command()
+@click.option("-x", "--xml-path")
+@click.option("-f", "--func-path")
+@click.option("-t", "--targ-path")
+def information(xml_path, func_path, targ_path):
+    """Enter the information needed to generate your package."""
+    create_package(xml_path, func_path, targ_path)
```
Comment (RobPasMue) [inline src/pydita_ast/cli.py:93]: Why not just "package" instead of having to do "package information"?

Comment (clatapie) [reply src/pydita_ast/cli.py]: You're right, I made the changes

File: .github/workflows/autodoc_cicd.yml [hunk 3]
```diff
@@ -55,7 +55,7 @@ jobs:
       
       - name: "Create pydita-autogenerated doc"
         run: |
-          python ./xml2rst.py -p ${{ github.workspace }}/mapdl-cmd-doc -f ${{ github.workspace }}/tests/customized_functions
+         pydita-ast package -x ${{ github.workspace }}/mapdl-cmd-doc -f ${{ github.workspace }}/tests/customized_functions
```
Comment (RobPasMue) [inline .github/workflows/autodoc_cicd.yml:58]:
```suggestion
          pydita-ast package -x ${{ github.workspace }}/mapdl-cmd-doc -f ${{ github.workspace }}/tests/customized_functions
```

**Pull request merged**